ci/fireci/fireci/ci_utils.py

```diff
@@ -61,3 +63,9 @@ def gcloud_identity_token():
   """Returns an identity token with the current gcloud service account."""
   result = subprocess.run(['gcloud', 'auth', 'print-identity-token'], stdout=subprocess.PIPE, check=True)
   return result.stdout.decode('utf-8').strip()
+
+def get_projects(file_path: str = "subprojects.cfg") -> List[str]:
+  """Parses the specified file for a list of projects in the repo."""
+  with open(file_path, 'r') as file:
```

Comment:
You can use the config parser module for this

```
import configparser

config = configparser.ConfigParser(delimiters=('='), allow_unnamed_section=True, allow_no_value=True)
config.read('subprojects.cfg')
return [k for k,v in config.items(configparser.UNNAMED_SECTION)]
```

Reply:
Ouch, 3.9 is old, we should bump that... but not all the way to 3.13.

Feel free to ignore the suggestion.